### PR TITLE
Add covenant committe info and global params

### PR DESCRIPTION
**Pull request by KonradStaniec**
- Add Covenant committee info
- Add Mainnet global parameters

### CODE REVIEW

File: bbn-1/covenant-committee/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+The Covenant Emulation Committee comprises entities operating
+[Covenant Signer](https://github.com/babylonlabs-io/covenant-signer) daemons.
+
+For this Mainnet, the Covenant Emulation Committee has 9 members, 3 of which
+are operated by the Babylon Foundation. The Covenant quorum configuration as
+well as the current members can be found in the
+[staking parameters](../parameters/global-params.json).
+
+A full list of the endpoints follows below, grouped by the operating entity.
```
Comment (vitsalis) [inline bbn-1/covenant-committee/README.md:11]: Let's also list the public key corresponding to each covenant signer?

**Review by vitsalis — APPROVED** [review]
LFG 🚀

**Review by filippos47 — APPROVED** [review]

File: bbn-1/covenant-committee/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Babylon Foundation
```
Comment (vitsalis) [inline bbn-1/covenant-committee/README.md:13]:
```suggestion
## Babylon Labs

Babylon labs maintains 3 signers with the following keys and URLs.
```

File: bbn-1/covenant-committee/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+- key-0: 03d45c70d28f169e1f0c7f4a78e2bc73497afe585b70aa897955989068f3350aaa
```
Comment (vitsalis) [inline bbn-1/covenant-committee/README.md:15]: Let's wrap keys with backticks

File: bbn-1/covenant-committee/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- url-0: https://covenant-signer0.babylonlabs.io
+- key-1: 034b15848e495a3a62283daaadb3f458a00859fe48e321f0121ebabbdd6698f9fa
+- url-1: https://covenant-signer1.babylonlabs.io
+- key-2: 0223b29f89b45f4af41588dcaf0ca572ada32872a88224f311373917f1b37d08d1
+- url-2: https://covenant-signer2.babylonlabs.io
```
Comment (vitsalis) [inline bbn-1/covenant-committee/README.md:20]:
```suggestion
- Signer 0
   - key: 03d45c70d28f169e1f0c7f4a78e2bc73497afe585b70aa897955989068f3350aaa
   - URL: https://covenant-signer0.babylonlabs.io
- Signer 1
   - key: 034b15848e495a3a62283daaadb3f458a00859fe48e321f0121ebabbdd6698f9fa
   - URL: https://covenant-signer1.babylonlabs.io
- Signer 2
   - key: 0223b29f89b45f4af41588dcaf0ca572ada32872a88224f311373917f1b37d08d1
   - URL: https://covenant-signer2.babylonlabs.io
```

**Review by filippos47 — DISMISSED** [review]

**Review by filippos47 — APPROVED** [review]

File: bbn-1/covenant-committee/README.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+# Covenant Committee
+
+The Covenant Emulation Committee comprises entities operating
+[Covenant Signer](https://github.com/babylonlabs-io/covenant-signer) daemons.
+
+For this Mainnet, the Covenant Emulation Committee has 9 members, 3 of which
+are operated by the Babylon Foundation. The Covenant quorum configuration as
```
Comment (vitsalis) [inline bbn-1/covenant-committee/README.md:7]:
```suggestion
are operated by Babylon Labs. The Covenant quorum configuration as
```

**Review by vitsalis — APPROVED** [review]

**Review by vitsalis — APPROVED** [review]

**Review by filippos47 — APPROVED** [review]

**Comment by antonio-altr** [comment]
AltLayer params downloaded and applied

**Comment by liewhite** [comment]
CoinSummer Labs params downloaded and applied

**Comment by kwantam** [comment]
Cubist prarams downloaded and applied

**Comment by AlexRockX** [comment]
RockX params downloaded and applied

**Comment by zellic-audit** [comment]
Zellic params downloaded and applied

**Pull request merged**